tests/test_regressions.py

```diff
@@ -3266,7 +3266,7 @@ def view_func(request, format=None):
 @mock.patch('drf_spectacular.settings.spectacular_settings.OAS_VERSION', '3.1.0')
 def test_basic_oas_3_1_nullable_cases(no_warnings, django_transforms):
     class M14(models.Model):
-        field_json = models.JSONField(null=True)  # case 1
+        field_json = models.JSONField(null=True, help_text="field_json desc")  # case 1
```

Comment:
would you mind adding a new field instead of changing the existing one. we'll get more coverage that way.

Reply:
Thanks for fixing it, hadn't had time to do it